### PR TITLE
feat(pubsublite): publish and subscribe samples for Pub/Sub Lite

### PR DESCRIPTION
**Pull request by tmdiep**
Adds samples for publishing and subscribing using Pub/Sub Lite.

### CODE REVIEW

**Comment by tmdiep** [comment]
@hongalex Can you please review these. I haven't been granted access to this repo and can't add reviewers. Thanks!

File: pubsublite/publish/publish_batch.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"cloud.google.com/go/pubsublite/pscompat"
+)
+
+func publishWithBatchSettings(w io.Writer, projectID, zone, topicID string, messageCount int) error {
+	// projectID := "my-project-id"
+	// zone := "us-central1-a"
+	// topicID := "my-topic"
```
Comment (hongalex) [inline pubsublite/publish/publish_batch.go:31]:
```suggestion
	// topicID := "my-topic"
	// messageCount := 10
```

File: pubsublite/publish/publish_ordering_key.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	"io"
+
+	"cloud.google.com/go/pubsub"
+	"cloud.google.com/go/pubsublite/pscompat"
+)
+
+func publishWithOrderingKey(w io.Writer, projectID, zone, topicID string, messageCount int) error {
+	// projectID := "my-project-id"
+	// zone := "us-central1-a"
+	// topicID := "my-topic"
```
Comment (hongalex) [inline pubsublite/publish/publish_ordering_key.go:30]:
```suggestion
	// topicID := "my-topic"
	// messageCount := 10
```

File: pubsublite/publish/publish_batch.go (hunk 1 above)
Comment (tmdiep) [reply pubsublite/publish/publish_batch.go:31]: Good catch, thanks! Done.

**Review by tbpg — COMMENTED** [review]
A few small comments. LGTM overall.

File: pubsublite/publish/publish_test.go [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+func TestPublish(t *testing.T) {
+	ctx := context.Background()
+	admin, err := pubsublite.NewAdminClient(context.Background(), region)
+	if err != nil {
+		t.Fatalf("pubsublite.NewAdminClient: %v", err)
+	}
+	defer admin.Close()
+
+	tc := testutil.SystemTest(t)
+	test.Cleanup(t, admin, tc.ProjectID, []string{zone})
```
[the remark this answers is not in the document]
Comment (tmdiep) [reply pubsublite/publish/publish_test.go]: Good point. I renamed to `psltest`.

File: pubsublite/publish/publish_test.go [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+func TestPublish(t *testing.T) {
+	ctx := context.Background()
+	admin, err := pubsublite.NewAdminClient(context.Background(), region)
+	if err != nil {
+		t.Fatalf("pubsublite.NewAdminClient: %v", err)
+	}
+	defer admin.Close()
+
+	tc := testutil.SystemTest(t)
```
[the remark this answers is not in the document]
Comment (tmdiep) [reply pubsublite/publish/publish_test.go]: Done

File: pubsublite/internal/test/util.go [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+	"testing"
+
+	"cloud.google.com/go/pubsublite"
+	"google.golang.org/api/iterator"
+)
+
+// Cleanup deletes all previous test topics/subscriptions from previous test
+// runs. This prevents previous test failures from building up resources that
+// count against quota.
+func Cleanup(t *testing.T, client *pubsublite.AdminClient, proj string, zones []string) {
```
[the remark this answers is not in the document]
Comment (tmdiep) [reply pubsublite/internal/test/util.go]: This is just some standalone cleanup and unrelated to a particular test, so I think the line numbers in this function are more suitable.

**Comment by tmdiep** [comment]
@tbpg @hongalex Thanks for the review! All comments addressed.

**Review by tbpg — APPROVED** [review]

**Review by hongalex — APPROVED** [review]

**Pull request merged**